src/app/admin/requests/request-details/request-details.component.html

```diff
@@ -1,22 +1,295 @@
-<div class="mt-4">
-  <a href="#" appGoBack>&lt;- Înapoi</a>
+<div class="modal_header_line" fxLayout="row" fxLayoutAlign="center center">
+  <a>Crează Cerere Nouă</a>
```

Comment:
Why page for showing request details, was refactored into page to create request?